src/complex/Utilities/DataArrayUtilities.hpp

```diff
@@ -153,7 +153,7 @@ struct ConvertTo<bool>
       return {floatResult.value() != 0.0};
     }
 
-    return {false};
+    return {true};
```

Comment:
> What was the reason this was changed, and is this used somewhere that it could break assumptions should the input not have a clear conversion?

I had hopped on a call with Mike and he told me that he wants boolean values to always return true if they are not explicitly false and he approved the change in advance so long as all the affected classes tests still passed.

Reply:
Alright. Sounds good to me.